admin/sql/create_tables.sql

```diff
@@ -5,25 +5,45 @@ CREATE TABLE lowlevel (
   mbid        UUID NOT NULL,
   build_sha1  TEXT NOT NULL,
   lossless    BOOLEAN                  DEFAULT 'n',
-  data        JSON NOT NULL,
-  submitted   TIMESTAMP WITH TIME ZONE DEFAULT NOW(),
-  data_sha256 TEXT NOT NULL
+  submitted   TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+);
+
+CREATE TABLE lowlevel_json (
+  id          INTEGER, -- FK to lowlevel.id
+  data        JSONB    NOT NULL,
+  data_sha256 CHAR(64) NOT NULL
 );
 
 CREATE TABLE highlevel (
   id         INTEGER, -- FK to lowlevel.id
   mbid       UUID    NOT NULL,
   build_sha1 TEXT    NOT NULL,
-  data       INTEGER NOT NULL, -- FK to highlevel_json.data
   submitted  TIMESTAMP WITH TIME ZONE DEFAULT NOW()
 );
 
-CREATE TABLE highlevel_json (
-  id          SERIAL,
-  data        JSON     NOT NULL,
+CREATE TABLE highlevel_meta (
+  id          INTEGER, -- FK to highlevel.id
+  data        JSONB    NOT NULL,
   data_sha256 CHAR(64) NOT NULL
 );
 
+CREATE TABLE highlevel_model (
+  id          SERIAL,
+  highlevel   INTEGER, -- FK to highlevel.id
+  data        JSONB    NOT NULL,
+  data_sha256 CHAR(64) NOT NULL,
+  model       INTEGER -- FK to model.id
+);
+
+CREATE TABLE model (
+  id            SERIAL,
+  model         TEXT NOT NULL,
+  model_version TEXT NOT NULL,
+  build_sha1    TEXT NOT NULL,
```

Comment:
What's in `model`, `model_version`, and `build_sha1`? It seems like version of gaia would be important in case of models, since that's what we use to generate them. And I agree that git revision should be more useful.